app/wasm/bindings/query.go

```diff
@@ -22,8 +22,8 @@ type OsmosisQuery struct {
 }
 
 type FullDenom struct {
-	Contract string `json:"contract"`
-	SubDenom string `json:"sub_denom"`
+	CreatorAddr string `json:"creator_addr"`
+	Subdenom    string `json:"subdenom"`
```

Comment:
Why is this `Subdenom` and not `SubDenom`?

Reply:
subdenom seems like a better name right? We say subroutine not subRoutine, or sub_routine